src/org/usfirst/frc/team467/robot/Autonomous/ActionGroup.java

```diff
@@ -180,4 +183,43 @@ public ReachAngle(double rotationInDegrees) {
 	public String getName() {
 		return name;
 	}
+	static class ConcurrentActions implements Action.Activity{
+		List<Action.Activity> activities = new ArrayList<Action.Activity>();
+		
+		public ConcurrentActions(Action.Activity... activities) {
+			this.activities = Arrays.asList(activities);			
+		}
+		
+		@Override
+		public void doIt() {
+			for(Action.Activity activity : activities) {
+				activity.doIt();
+				
+			}
+			
+		}	
+	}
+	static class multicondition implements Action.Condition{
```

Comment:
The class name should be MultiCondition. Also, it should be rewritten to use the varargs style used in ConcurrentActions.